pkgs/servers/misc/airsonic-advanced/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl, nixosTests }:
+
+stdenv.mkDerivation rec {
+  pname = "airsonic-advanced";
+  version = "11.0.0-SNAPSHOT.20211116202136";
+
+  src = fetchurl {
+    url = "https://github.com/airsonic-advanced/airsonic-advanced/releases/download/${version}/airsonic.war";
+    sha256 = "sha256-soLnjltoATjqNHRpcVcqiTSQGJwSSzxwKEMk/cr42YU=";
+  };
+
+  buildCommand = ''
```

```suggestion
  installPhase = ''
```
we are not building anything